terraform/shared/video-player/js/common.js

```diff
@@ -21,6 +21,42 @@ var live_ipfs = getURLParam('live') // Set m3u8 file URL to override IPFS live s
 var vod_ipfs = getURLParam('vod')   // Set IPFS content hash of mp4 file to play IPFS on-demand video stream
 var startFrom = getURLParam("startFrom"); // Timecode to start video playing from
 
+// If startFrom starts with a Q its probaly a IPFS hash so calculate to correct startFrom

```

Comment:
This is not a reliable way of detecting, I am not aware this is promised in CID spec. See: https://github.com/ipfs/ipfs/issues/337

We are probably better of checking for number vs. string. Alternative is to use two separate URL param keys.